index.js

```diff
@@ -14,6 +16,29 @@ program.parse();
 const options = program.opts();
 const AWS = "AWS";
 
+const calculateUnits = (result, provider) => {
+  const providerUnits = UNIT_MAPPING[provider];
+  let cspmUnits = 0;
+  let cwppUnits = 0;
+  Object.keys(result).forEach((serviceName) => {
+    if (serviceName === "total") {
+      return;
+    }
+    const serviceResourceTypesCounts = result[serviceName];
+    const serviceUnits = providerUnits[serviceName];
+    Object.keys(serviceResourceTypesCounts).forEach((resourceTypeCount) => {
+      if (serviceUnits[resourceTypeCount] !== undefined) {
+        cwppUnits +=
+          serviceUnits[resourceTypeCount] *
+          serviceResourceTypesCounts[resourceTypeCount];
+      } else {
+        cspmUnits += serviceResourceTypesCounts[resourceTypeCount];
+      }
```

Comment:
```suggestion
      cspmUnits += serviceResourceTypesCounts[resourceTypeCount];
```

AWS EC2, Lambda, and ECS will also be counted in the CSPM usage